Documentation/maintainers/regenerate-files

```diff
@@ -10,7 +10,7 @@ SCRIPT=$(basename $0)
 HEADER="# Autogenerated by ${SCRIPT}, DO NOT EDIT."
 
 KF_RELEASES="5.21 5.22"
-KF_RELEASE=${KF_RELEASE:-5.21}
+KF_RELEASE=${KF_RELEASE:-5.22}
```

Comment:
KDE Framworks 5.21 is stable 5.22 not. So why should the default sets be the unstable version?